docs/MAINTAINERS.md

```diff
@@ -0,0 +1,42 @@
+## For maintainers
+
+### Helper script for creating fortran tests: create\_fortran\_test.py
+
+A easy way to create an exercise test is to use the script
+bin/create\_fortran\_test.py
+
+Use this script to create an initial <exercise>\_test.f90 file which can be used as a skeleton for your test.
+Typically, you will have to replace the 'response'-function in the generated file with the correct function call.
+
+Also note that Fortran has issues with special characters such as `\n` and `\t` so take special care handling these.
+
+#### Prerequsites
+- Working cmake and fortran compiler
+- Python3.x (You can make it may work with Python2, but I have not made the
+effort to make it backwards compatible)
+- latest version of https://github.com/exercism/problem-specifications.git
+
+#### Work flow for creating a new test
+- pull latest changes from exercism/problem-specifications
+- run this script for the example you want to create
+- copy config/CMakeLists.txt for exercise directory
+- implement working exercise
+- fix potential problematic tests (see eg. exercise/bob "Test 20" and "Test 24")
+- ensure ctest validates without errors
```

Comment:
Use black ticks when referencing the command. `ctest`

Reply:
resolved by https://github.com/exercism/fortran/pull/63/commits/cfe98e38744ca93d8eb3491483789cbd6ea30faf